frontend/src/components/SyncEditor/process.ts

```diff
@@ -118,10 +119,34 @@ export const processUser = (
   readonly: boolean,
   validators: any,
   currentEditorValue: string,
-  editableUidSiblings: boolean | undefined
+  editableUidSiblings: boolean | undefined,
+  autoCreateNs?: boolean
 ) => {
   // get yaml, documents, resource, mapped
-  const documents: any[] = YAML.parseAllDocuments(yaml, { prettyErrors: true, keepCstNodes: true })
+  let documents: any[] = YAML.parseAllDocuments(yaml, { prettyErrors: true, keepCstNodes: true })
+  let createNs = false
+  if (autoCreateNs) {
+    const newDocs = documents.map((doc: YAML.Document) => {
+      if (doc.has('metadata') && !doc.hasIn(['metadata', 'namespace'])) {
+        createNs = true
+        const pair = new Pair('namespace', '')
```

Comment:
I'm wondering if the default should be `open-cluster-management-global-set` instead. What do you think @JustinKuli, @dhaiducek, and @gparvin ?

Reply:
Does it currently set it to an empty string, and then presumably refuse the request / warn the user if they try to apply it as-is? I think that in some way, it should tell the user that they need to set the namespace. I don't really think we should "recommend" `open-cluster-management-global-set`.

Reply:
`open-cluster-management-global-set` is really too long to use as a default I think.  There is another concern regarding using ACM namespaces as a default policy namespace too.  This namespace can be deleted in a partial uninstall scenario that keeps CRDs and would leave policy CRs (except in ACM namespaces).  If it was a policy we provide out of the box or through our automation it would be more acceptable.